sources/core/Stride.Core.CompilerServices/Analyzers/STRDIAG006InvalidAssignMode.cs

```diff
@@ -0,0 +1,82 @@
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using System.Diagnostics;
+using Stride.Core.CompilerServices.Common;
+
+namespace Stride.Core.CompilerServices.Analyzers;
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public class DataMemberModeAnalyzer : DiagnosticAnalyzer
```

Comment:
Rename class to match file name

Reply:
[d364af7](https://github.com/stride3d/stride/pull/1864/commits/d364af77f4f17fac1d41c8d111d79542b76a00c0)

renamed it